k8s/demo/common/rd/professional-api.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: rd-professional-api
-    version: 0.0.7
+    version: 0.0.8
```

Comment:
Chart version 0.0.8 does not exist. 0.0.7 is the latest available version. Please revert this change.